lib/core/presentation/pages/chat/chat_message/widgets/chat_input/reply_content_widget.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+
+class ReplyContent extends StatelessWidget {
+  final Event replyEvent;
+  final bool ownMessage;
+  final Timeline? timeline;
+
+  const ReplyContent(
+    this.replyEvent, {
+    this.ownMessage = false,
+    Key? key,
+    this.timeline,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    Widget replyBody;
+    final timeline = this.timeline;
+    final displayEvent =
+        timeline != null ? replyEvent.getDisplayEvent(timeline) : replyEvent;
+
+    replyBody = Text(
+      displayEvent.calcLocalizedBodyFallback(
+        MatrixDefaultLocalizations(),
```

Comment:
We only use `repyBody` once, no need to declare it like this. If you must separate for clarification, but it in another class in another file.